test/src/main/java/org/apache/accumulo/test/functional/MemoryStarvedMinCIT.java

```diff
@@ -87,7 +87,7 @@ public static void start() throws Exception {
           }
           if (line.startsWith("accumulo")) {
             Metric metric = TestStatsDSink.parseStatsDMetric(line);
-            if (MINC_PAUSED.getName().equals(metric.getName())) {
+            if (MAJC_PAUSED.getName().equals(metric.getName())) {
```

Comment:
This was correct before

Reply:
Thanks for taking a close look at everything. Making a conversion like this is very error prone, evidently.